coala_quickstart/info_extractors/RequirementsInfoExtractor.py

```diff
@@ -0,0 +1,33 @@
+from requirements import parse
```

Comment:
Make sure to add the corresponding packages to `requirements.txt`. Builds are failing due to this :(